packages/menu/src/Menu.ts

```diff
@@ -256,18 +427,10 @@ export class Menu extends SpectrumElement {
     public connectedCallback(): void {
         super.connectedCallback();
         if (!this.hasAttribute('role')) {
-            const queryRoleEvent = new CustomEvent('sp-menu-query-role', {
-                bubbles: true,
-                composed: true,
-                detail: {
-                    role: '',
-                },
-            });
-            this.dispatchEvent(queryRoleEvent);
-            this.setAttribute('role', queryRoleEvent.detail.role || 'menu');
+            this.setAttribute('role', 'menu');
```

Comment:
Should this sometimes be `group`? How?

Reply:
Looks like I didn't finish the cleanup here. I just set the role directly in `PickerBase` now. That's the only place we were listening to this event internally.